packages/ckeditor5-widget/src/widget.ts

```diff
@@ -630,9 +655,21 @@ function isInsideNestedEditable( element: ViewElement ) {
 		currentElement = currentElement.parent;
 	}
 
+	/* istanbul ignore next -- @preserve */
 	return false;
 }
 
```

Comment:
It looks like incorrect `istanbul` report, because `return false;` is reached during the tests.